packages/inter-protocol/test/amm/vpool-xyk-amm/test-amm-governance.js

```diff
@@ -40,7 +40,7 @@ test('start Economic Committee', async t => {
   t.truthy(creator);
 });
 
-test('amm change param via Governance', async t => {
+test.only('amm change param via Governance', async t => {
```

Comment:
☝️

Reply:
I don't know why my pre-submit pattern doesn't catch this one.  It routinely tells me about test.only in other places, but it continues to not see this one.